### PR TITLE
Fix/window is not found

### PR DESCRIPTION
**Pull request by asam139**
Fix bug: get first window to get screenshot

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

In some cases and devices the SnapshotHelper.swift fails finding the window using the current way.

I have tested with all simulators for iOS 10 and iOS 11 (iPad an iPhone)

### Description

Change the way which the window is obtained.

I saw an old pull request (merged) which this line was modified. I have a couple of apps where the current code does not work. Getting the first window works really fine.

### CODE REVIEW

**Comment by joshdholtz** [comment]
I'm down for this change. This actually is how Apple recommends on it - https://developer.apple.com/documentation/xctest/activities_and_attachments/adding_attachments_to_tests_and_activities

screen shot 2018-07-28 at 1 02 10 pm

**Comment by asam139** [comment]
Yes it is. I saw it in the documentation, so how you prefer it. :). 

Using "firstMatch" works really fine for me, the other way alway fails finding the window.

**Comment by asam139** [comment]
I have updated the version to 1.12 in case that pull request is merged 👍 
If not I keep my fork jajaja

Thanks so much.

**Comment by joshdholtz** [comment]
@asam139 I was just gunna ask if you could bump that version 😉 Thanks!

File: snapshot/lib/assets/SnapshotHelper.swift [hunk 1]
```diff
@@ -160,11 +160,7 @@ open class Snapshot: NSObject {
                 return
             }
             
-            guard let window = app.windows.allElementsBoundByIndex.first(where: { $0.frame.isEmpty == false }) else {
-                print("Couldn't find an element window in XCUIApplication with a non-empty frame.")
-                return
-            }
-
+            let window = window.firstMatch
```
Comment (joshdholtz) [inline snapshot/lib/assets/SnapshotHelper.swift]: I _think_ this line should really be `let window = app.windows.firstMatch`?

**Review by joshdholtz — CHANGES_REQUESTED** [review]

File: snapshot/lib/assets/SnapshotHelper.swift (hunk 1 above)
Comment (asam139) [reply snapshot/lib/assets/SnapshotHelper.swift]: Yes it is, sorry, copy and paste is so dangerous jejeje. I have it fixed.

**Review by joshdholtz — APPROVED** [review]
Works for me 💪

**Comment by owurman** [comment]
Unfortunately, this actually **reverted** the fix for #12174, so that's now failing in some cases again.

Can we restore that fix, and instead of returning an error, fall back on firstMatch?

**Pull request merged**